src/org/usfirst/frc2876/Steamworks2017/commands/AutoMobility.java

```diff
@@ -0,0 +1,67 @@
+package org.usfirst.frc2876.Steamworks2017.commands;
+
+import org.usfirst.frc2876.Steamworks2017.Robot;
+
+import edu.wpi.first.wpilibj.Timer;
+import edu.wpi.first.wpilibj.command.Command;
+
+public class AutoMobility extends Command {

```

Comment:
I think the name "Mobility" is coming from the game manual.  Section 4.3 refers to the various autonomous point values, and refers to the movement as the "Mobility" bonus.  I think they decided to put the code in here because they were testing the conditions above (distance PID working) but it was supposed to do the same thing (nearly) either way.

By the way, I am answering here because I was with them on Saturday when they wrote it, and although the commits are sometimes in my name (or rwodonnell - still not sure why), it was entirely written by the students.